content/docs/forms.md

```diff
@@ -182,13 +182,49 @@ class FlavorForm extends React.Component {
 
 Overall, this makes it so that `<input type="text">`, `<textarea>`, and `<select>` all work very similarly - they all accept a `value` attribute that you can use to implement a controlled component.
 
-> Note
->
-> You can pass an array into the `value` attribute, allowing you to select multiple options in a `select` tag:
->
->```js
-><select multiple={true} value={['B', 'C']}>
->```
+### The select multiple Tag
+Just like in HTML, select elements can become multi-select elements through the addition of the *multiple* attribute to the select tag. But handling of the value attribute in react is more complex due to having an array of values. 
```

Comment:
```suggestion
Just like in HTML, select elements can become multi-select elements through the addition of the *multiple* attribute to the select tag. In React, the `value` attribute for `multiple` is handled as an array. 
```

Reply:
I see where you're going with that but I don't like your wording of it either. How about: 
> In React, this results in the `value` attribute being handled as an array of selected values, rather than just one value.